webview/src/setup/components/remotes/ShowExtension.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styles from './styles.module.scss'
+import { Icon } from '../../../shared/components/Icon'
+import { Extensions } from '../../../shared/components/icons'
+
+export const ShowExtension: React.FC<{ id: string; name: string }> = ({
+  id,
+  name
+}) => {
+  const idQuery = `"@id:${id}"`
+
+  return (
+    <p>
+      <Icon
```

Comment:
With this admonition added, there are now 2-3 admonitions stacked up on top of each other in each panel:

<img width="506" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/a06812bc-30c3-4872-9684-158d888f71e6">

This feels hard to read for me but I'm not really seeing an alternative solution besides rewording these into paragraphs instead of admonitions which might not actually make these more readable 🤔